docs/config-file-resolutions-yml.md

```diff
@@ -6,7 +6,7 @@ by providing a reason why they are acceptable and can be ignored.
 You can use the [resolutions.yml example](../examples/resolutions.yml) as the base configuration file for your scans.
 
 ### When to Use Resolutions
-Resolutions should be used when it is impossible to solve an issue or a fix is planned for a later time.
+Resolutions should be used when it is impossible to solve an issue, or when a fix is planned for a later time.
```

Comment:
For a similar reason as stated above, I believe we must not use a comma here.

Reply:
Right. Done.